src/pages/references/stacking-contract.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Stacking Contract
+description: See a detailed list of all functions and error codes of the Stacking contract.
+---
+
+export { convertStackingRefToMdx as getStaticProps } from '@common/data/stacking-ref'
+import { StackingErrorcodeReference, StackingFunctionReference } from '@components/stacking-ref'
+
+Stacking is implemented as a smart contract using Clarity. On the testnet, you can always find the Stacking contract by this contract identifier: `ST000000000000000000002AMW42H.pox`.
```

Comment:
We need to add some kind of h2 here so that this paragraph gets wrapped in a section, and as a result gets styled correctly. `Introduction` or `Overview` would work.